plugins/woocommerce/src/Internal/ReceiptRendering/ReceiptRenderingEngine.php

```diff
@@ -381,17 +383,19 @@ private function get_order_data( WC_Order $order ): array {
 			),
 			'texts'            => array(
 				'receipt_title'                => $receipt_title,
-				'amount_paid_section_title'    => __( 'Amount Paid', 'woocommerce' ),
-				'date_paid_section_title'      => __( 'Date Paid', 'woocommerce' ),
+				'amount_paid_section_title'    => $is_order_paid ? __( 'Amount Paid', 'woocommerce' ) : __( 'Order Total', 'woocommerce' ),
+				'date_paid_section_title'      => $is_order_paid ? __( 'Date Paid', 'woocommerce' ) : __( 'Order Date', 'woocommerce' ),
 				'payment_method_section_title' => __( 'Payment method', 'woocommerce' ),
+				'order_status_section_title'   => __( 'Order status', 'woocommerce' ),
+				'order_status'                 => wc_get_order_status_name( $order->get_status() ),
```

Comment:
The one concern I have with this is that some merchants might have an order workflow where it goes to "processing" or "on hold" or something custom after the payment is successfully made, which they probably wouldn't want showing on their receipts.

🤔 Maybe the section title should be "Payment Status" instead of "Order Status", and then we would just have "Failed" or "Success" as the two possible values?